gcloud-java-dns/src/main/java/com/google/gcloud/dns/ChangeRequest.java

```diff
@@ -148,10 +148,41 @@ public Dns dns() {
   }
 
   /**
-   * Applies this change request to the associated zone.
+   * Applies this change request to the zone identified by {@code zoneName}.
    */
-  public ChangeRequest applyTo(Dns.ChangeRequestOption... options) {
-    return dns.applyChangeRequest(zone, this, options);
+  public ChangeRequest applyTo(String zoneName, Dns.ChangeRequestOption... options) {
+    return dns.applyChangeRequest(zoneName, this, options);
+  }
+
+  /**
+   * Retrieves the up-to-date information about the change request from Google Cloud DNS. Parameter
+   * {@code options} can be used to restrict the fields to be included in the updated object the
+   * same way as in {@link Dns#getChangeRequest(String, String, Dns.ChangeRequestOption...)}. If
+   * {@code options} are provided, any field other than generatedId which is not included in the
+   * {@code options} will be {@code null} regardless of whether they are initialized or not in
+   * {@code this} instance.
+   *
+   * @return an object with the updated information or {@code null} if it does not exist
+   * @throws DnsException upon failure of the API call or if the associated zone was not found
+   */
+  public ChangeRequest reload(Dns.ChangeRequestOption... options) {
+    return dns.getChangeRequest(zone, generatedId(), options);
+  }
+
+  /**
+   * Returns {@code true} if the change request has been completed. The function makes an API call
+   * to Google Cloud DNS in order to request the status update only if the status of the change
+   * request is {@link Status#PENDING}. If the status is already {@link Status#DONE}, the method
```

Comment:
I think this could be made more concise suggesting something like "This call will make an API call if status is not complete."